pkg/client/deploymentlogs.go

```diff
@@ -34,15 +35,27 @@ func newDeploymentLogs(c *Client, namespace string) *deploymentLogs {
 
 // Get gets the deploymentlogs and return a deploymentLog request
 func (c *deploymentLogs) Get(name string, opts api.DeploymentLogOptions) *kclient.Request {
-	req := c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("log")
+	req := c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("log").
+		Param("follow", strconv.FormatBool(opts.Follow)).
+		Param("timestamps", strconv.FormatBool(opts.Timestamps))
```

Comment:
> Yes, there is pkg/conversion/queryparams.Convert but it only works against versioned objects with tags. opts here is our internal and doesn't have tags...

Open an issue in kube and in origin to fix the serialization of objects to query parameters and add TODOs in our code here.   This is acting as an unversioned codec, an unversioned defaulter, and a repetition of intent.